Giancarix117

Giancarix117 opened the pull request:
Homework for index

Giancarix117 committed:
Create giancarlo.html

intial commit

Giancarix117 committed:
Section 1

Commit for section 1

Giancarix117 committed:
Section 2

Commits for section 2

Giancarix117 committed:
Section 3

Commit for section 3

Giancarix117 committed:
Section 5

Commit for section 5

Giancarix117 committed:
Section 6

Changes section 6

Giancarix117 committed:
Section 6

Yeah so apparently at some point in my life i forgot how to count and decided that the number 4 doesnt actually exist. So this is the actual section 6.

Giancarix117 committed:
Section 7

Commits for section 7

Giancarix117 committed:
Section 10

Commits for Section 10

Giancarix117 committed:
Section 10 like the acctual one

Apparently i messed up a lot and like a lot of things got messed up because of it. Anyways if you are reading this just ignore the numbers on the other commits. I fixed a bunch of stuff i messed up whenever i was copying and pasting it from visual studio.  bunch of stuff also got cut off. This commit fixes everything

Giancarix117 committed:
Commit 11

this one is fine

Giancarix117 committed:
Section 12

And thats a wrap :D

iamcucusa reviewed (changes requested):
Hello, you still need to validate your code using the w3c validator. I think you still have around
20 errors.
Please review and add those changes.
well done so far

Giancarix117 committed:
Final version of giancarlo.html

Made the corrections

Giancarix117 reviewed (commented):
1. The final file is giancarlo-corrections.html

iamcucusa commented:
> 1. The final file is giancarlo-corrections.html

Hello, I see but fix the errors it is not to copy and paste a new file. Please fix the errors in. a way that they can be checked in the commits, on your file, not a new one.

Giancarix117 commented:
Yeah i fixed the errors, but honestly I didn’t link the VScode file to github, so i was manually copy pasting everything from the editor into git hub. I fixed the errors, but instead of copy pasting everything into the old file, it was easier to make a new one. I guess i was a little lazy jajaja, but the new one has no errors

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Grace ***@***.***>
Sent: Friday, November 12, 2021 1:57:48 PM
To: iamcucusa/html-exercises ***@***.***>
Cc: Giancarix117 ***@***.***>; Author ***@***.***>
Subject: Re: [iamcucusa/html-exercises] Giancarix117 (PR #21)


  1.  The final file is giancarlo-corrections.html

Hello, I see but fix the errors it is not to copy and paste a new file. Please fix the errors in. a way that they can be checked in the commits, on your file, not a new one.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/iamcucusa/html-exercises/pull/21#issuecomment-967099948>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ANR247EJ54VPJODM2XJWDFTULUFMZANCNFSM5H2U3ELA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

iamcucusa commented:
> Yeah i fixed the errors, but honestly I didn’t link the VScode file to github, so i was manually copy pasting everything from the editor into git hub. I fixed the errors, but instead of copy pasting everything into the old file, it was easier to make a new one. I guess i was a little lazy jajaja, but the new one has no errors Get Outlook for iOS<https://aka.ms/o0ukef>
> […](#)
> ________________________________ From: Grace ***@***.***> Sent: Friday, November 12, 2021 1:57:48 PM To: iamcucusa/html-exercises ***@***.***> Cc: Giancarix117 ***@***.***>; Author ***@***.***> Subject: Re: [iamcucusa/html-exercises] Giancarix117 (PR #21) 1. The final file is giancarlo-corrections.html Hello, I see but fix the errors it is not to copy and paste a new file. Please fix the errors in. a way that they can be checked in the commits, on your file, not a new one. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub<[#21 (comment)](https://github.com/iamcucusa/html-exercises/pull/21#issuecomment-967099948)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ANR247EJ54VPJODM2XJWDFTULUFMZANCNFSM5H2U3ELA>. Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

You are too lazy, I am going to approve but keep in mind it is not like the rest of the students. Please do it as specified next time.

iamcucusa reviewed (approved):
Approving with two files.
